### ISSUE TITLE
Is WebGL platform support table current?

### ISSUE THREAD

**Issue by donmccurdy** [KhronosGroup/3D-Formats-Guidelines#17]
As of [June 2021](https://twitter.com/grorgwork/status/1402307966817116161) — 

> WebGL 2 is enabled in yesterday's iOS/macOS/etc Safari releases. Also, everything is now running on top of Metal via ANGLE.

The platform support table says "Use of Metal API is required." for formats like ETC2 and ASTC, and explains how to run Chromium with flags enabled. Should those formats work without flags in Safari at this point? They certainly don't work on my device (M1 chip) but I can't figure out what is missing, or what issue to track, if Safari has already moved to Metal+ANGLE and this texture support is still missing.

**Comment by lexaknyazev** [comment]
**tl;dr**: That section is current.

Although Safari is using ANGLE with Metal, it's not the same ANGLE as used in Chrome. The work on converging Safari's fork is still in progress. Eventually, both browser engines would use the same ANGLE source that has full support for all compressed texture formats.

**Comment by donmccurdy** [comment]
Thank you! That's helpful to know.

Some cross-references for future readers:

- https://github.com/donmccurdy/glTF-Transform/issues/446
- https://github.com/google/model-viewer/issues/3024
- https://github.com/mrdoob/three.js/issues/22899
